### PR TITLE
Add to flag another parsing

### PR DESCRIPTION
**Pull request by yoavrotems**
add a new flag parsing:
flag + `\s*=\s*(.*)`,

### CODE REVIEW

**Comment by yoavrotems** [comment]
added a new one because only change the old one the have 0 or 1 (add ? after " to the parse) or even more occurrence could make weird situations

**Comment by yoavrotems** [comment]
mentioned in https://github.com/aquasecurity/bench-common/issues/53

**Review by robertojrojas — APPROVED** [review]
LGTM

File: auditeval/test.go [hunk 1]
```diff
@@ -159,6 +159,7 @@ func getFlagValue(s, flag string) string {
 	var flagVal string
 	pttns := []string{
 		flag + `\s*=\s*"(.*)"`,
+		flag + `\s*=\s*(.*)`,
```
Comment (lizrice) [inline auditeval/test.go]:
```suggestion
		flag + `\s*=\s*([^\s]*)`,
```
This should capture anything after the = and optional whitespace, until there is some whitespace. I think it means the next line is unnecessary.

**Comment by lizrice** [comment]
I went ahead and did this change, see the last two commits. Also fixes #54.

The tests are currently failing on a problem that is fixed with #61. 
* @yoavrotems if you think I have the logic right in #61, please go ahead and hit the green button to merge it to master
* This branch should then need updating, and fingers crossed the tests will then pass and we can merge this one too

**Comment by lizrice** [comment]
@yoavrotems there was [one more change](https://github.com/aquasecurity/bench-common/pull/54/commits/924c4478644df06867f8b4c200191b5beb72e6f1) required

**Pull request merged**